RecoTracker/LSTCore/standalone/code/core/AccessHelper.cc

```diff
@@ -410,11 +410,11 @@ std::tuple<std::vector<unsigned int>, std::vector<unsigned int>> getHitIdxsAndHi
 // ==============
 
 //____________________________________________________________________________________________
-std::vector<unsigned int> getLSsFromTC(Event* event, unsigned int TC) {
+std::vector<unsigned int> getLSsFromTC(Event* event, unsigned int iTC) {
   // Get the type of the track candidate
-  TrackCandidates const* trackCandidates = event->getTrackCandidates().data();
-  short type = trackCandidates->trackCandidateType[TC];
-  unsigned int objidx = trackCandidates->directObjectIndices[TC];
+  auto const& trackCandidates = event->getTrackCandidates().const_view();
```

Comment:
Same `const&` comment. A few occurrences on this file and in `write_lst_ntuple.cc`.